ftw/testbrowser/core.py

```diff
@@ -206,7 +208,7 @@ def open_html(self, html):
         :type html: string or file-like object
         :returns: The browser object.
         """
-        self.response = self._load_html(html)
+        self.response = self._load_html(html, subparser=self.parser)
```

Comment:
If I understand it correctly, you want to achieve that the custom `self.parser` is always used.
When not just use `self.parser` in `self._load_html` instead of passing it around? It is on the same object.

By not passing it around we can make sure that we do not miss a place.
It is customiziable in the same way: by setting the attribute on a browser instance.